src/lib.rs

```diff
@@ -71,6 +72,7 @@ fn orjson(py: Python, m: &PyModule) -> PyResult<()> {
     m.add("OPT_SERIALIZE_NUMPY", encode::SERIALIZE_NUMPY)?;
     m.add("OPT_SERIALIZE_UUID", encode::SERIALIZE_UUID)?;
     m.add("OPT_SORT_KEYS", encode::SORT_KEYS)?;
+    m.add("OPT_PRETTY", encode::PRETTY)?;
```

Comment:
open to other option names - went with `PRETTY` because of serde's `to_writer_pretty`, but `INDENT` is another option that kind of matches the stdlib

Reply:
serde_json does support a different indent level one level deeper in the API.